pkgs/applications/networking/instant-messengers/pidgin-plugins/msn-pecan/default.nix

```diff
@@ -1,9 +1,10 @@
 { lib, stdenv, fetchurl, pidgin} :
 
-stdenv.mkDerivation {
-  name = "pidgin-msn-pecan-0.1.4";
+stdenv.mkDerivation rec {
+  pname = "pidgin-msn-pecan";
+  version = "0.1.4";
   src = fetchurl {
-    url = "http://msn-pecan.googlecode.com/files/msn-pecan-0.1.4.tar.bz2";
```

Comment:
Unrelated: both URLs 404.

Reply:
should we switch to: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/msn-pecan/msn-pecan-0.1.4.tar.bz2
or https://github.com/felipec/msn-pecan/releases/tag/v0.1.4 ?

Reply:
decided for the github link